python_ta/__init__.py

```diff
@@ -166,6 +166,24 @@ def _load_config(linter, config_location):
     linter.load_config_file()
 
 
+def _load_messages_config(path: str, def_path: str) -> dict:
+    """Given path (potentially) specified by user and default def_path
+    of messages config file, merge the config files."""
+    if def_path == path:
+        return toml.load(path)
+    merge_into = toml.load(def_path)
+    merge_from = toml.load(path)
+    for i in merge_from:
```

Comment:
This nesting is indicating some kind of assumed structure for the config file. Use meaningful variable names to convey what each level of structure means.